### ISSUE TITLE
[BUG] method `id()` has bug when  insert multiple tables

### ISSUE THREAD

**Issue by out001a** [catfan/Medoo#871]
**Information**
- Version of Medoo: latest
- Type of Database (MySQL, MSSQL, SQLite...): MySQL
- System (Liunx\Windows\Mac): Linux

**Detail Code**
```php
$database->insert("account",  $row1);    // insert success
$id1 = $database->id();

$database->insert("table_not_exist",  $row2);    // insert fail
$id2 = $database->id();
```

**Expected output**
`$id2` shoud be 0 or false, but actually its value is equal to `$id1`

**Comment by catfan** [comment]
Good spot. It will be fixed.

**Comment by catfan** [comment]
This bug have been fixed on v1.7.5.
